### ISSUE TITLE
ScrollDir is not defined

### ISSUE THREAD

**Issue by 0xAl3xH** [go-vgo/robotgo#606]
- Robotgo version (or commit ref): latest, `[v0.100.10](https://pkg.go.dev/github.com/go-vgo/robotgo@v0.100.10)`
- Go version: `go version go1.19.5 darwin/amd64`
- Gcc version: `Apple clang version 13.1.6 (clang-1316.0.21.2.5)`
- Operating system and bit: `macOS 12.3.1`
- Resolution: `3456 x 2234`
- Can you reproduce the bug at [Examples](https://github.com/go-vgo/robotgo/blob/master/examples/main.go):
  - [ ] Yes (provide example code)
  - [ ] No
  - [x ] Not relevant
- Provide example code:

```Go
package main

import (
  "github.com/go-vgo/robotgo"
)

func main() {
  robotgo.MouseSleep = 100

  robotgo.ScrollDir(10, "up")
  robotgo.ScrollDir(20, "right")

  robotgo.Scroll(0, -10)
  robotgo.Scroll(100, 0)

  robotgo.MilliSleep(100)
  robotgo.ScrollSmooth(-10, 6)
  // robotgo.ScrollRelative(10, -100)

  robotgo.Move(10, 20)
  robotgo.MoveRelative(0, -10)
  robotgo.DragSmooth(10, 10)

  robotgo.Click("wheelRight")
  robotgo.Click("left", true)
  robotgo.MoveSmooth(100, 200, 1.0, 10.0)

  robotgo.Toggle("left")
  robotgo.Toggle("left", "up")
}
```

## Description

The current example code uses `ScrollDir` which is not defined in v0.100.10 of this library which is the default installed version. The example code should force v1.x or take our `ScrollDir` entirely.

as a workaround, I forced `require github.com/go-vgo/robotgo v1.0.0-rc1` in `go.mod`

**Comment by timendez** [comment]
omg thank you so much! I forced that version too and it fixed my problem as well.

**Comment by vcaesar** [comment]
You should use this latest Released, and v0.100.10 is dropped now.

**Comment by timendez** [comment]
> You should use this latest Released, and v0.100.10 is dropped now.

Thank you! I thought I was, but at some point digging through logs it looked like VS Code automatically downloaded a very old version. I'm up to latest released now.
